R/loadReg2.R

```diff
@@ -444,14 +463,17 @@ predictSolute.loadReg2 <- function(
       }
     }
   }
-  
   # Add dates if requested
   if(date) {
     if(!is.data.frame(preds)) {
       preds <- data.frame(fit=preds)
     }
     # prepend the date column
-    preds <- data.frame(date=getCol(metadata, newdata, "date"), preds)
+    #preds <- data.frame(date=getCol(metadata, newdata, "date"), preds)
+    #predLoad returns the dates, don't need to get them from metadata
```

Comment:
are you sure? if we just created the data.frame in line 468 above, then preds will be a data.frame with only a `fit` column, no dates...

(however, if that's the only case when a date column needs to be added, it does look like these lines could be moved into the `if(!is.data.frame(preds))` block just above, right?)